IdealWeight.rb

```diff
@@ -0,0 +1,10 @@
+def calculate
+    puts "Enter Your Name"
+    name = gets.chomp
+    puts "Enter Your Height"
+    height = gets.chomp.to_i
+    weight = height - 110
```

Comment:
what is the magical number?
https://eliotsykes.com/2015/08/18/magic-numbers/
http://www.chrisrolle.com/en/blog/replacing-magic-numbers